rest/taskana-rest-spring/src/test/resources/application-ldap.properties

```diff
@@ -0,0 +1,51 @@
+logging.level.pro.taskana=INFO
+### logging.level.org.springframework=DEBUG
+######## Taskana DB #######
+datasource.url=jdbc:h2:mem:taskana;IGNORECASE=TRUE;LOCK_MODE=0
+datasource.driverClassName=org.h2.Driver
+datasource.username=sa
+datasource.password=sa
+taskana.schemaName=TASKANA
+####### property that control rest api security deploy use true for no security.
+devMode=false
+
+####### Property that informs about the Taskana's version. This version is shown the application web
+version=@project.version@
+
+####### control LDAP usage
+taskana.ldap.useLdap=true
+####### properties to connect to LDAP
+taskana.ldap.serverUrl=ldap://localhost:10389
+taskana.ldap.bindDn=uid=admin
+taskana.ldap.bindPassword=secret
+taskana.ldap.baseDn=ou=Test,O=TASKANA
+####### properties that control search for users and groups
+taskana.ldap.userSearchBase=cn=users
+taskana.ldap.userSearchFilterName=objectclass
+taskana.ldap.userSearchFilterValue=person
+taskana.ldap.userFirstnameAttribute=givenName
+taskana.ldap.userLastnameAttribute=sn
+taskana.ldap.userIdAttribute=uid
+taskana.ldap.groupSearchBase=cn=groups
+taskana.ldap.groupSearchFilterName=objectclass
+taskana.ldap.groupSearchFilterValue=groupOfUniqueNames
+taskana.ldap.groupNameAttribute=cn
+taskana.ldap.minSearchForLength=3
+taskana.ldap.maxNumberOfReturnedAccessIds=50
+taskana.ldap.groupsOfUser=memberUid
+####### JobScheduler cron expression that specifies when the JobSchedler runs
+taskana.jobscheduler.async.cron=0 0 * * * *
+####### cache static resources properties
+spring.resources.cache.cachecontrol.cache-private=true
+spring.main.allow-bean-definition-overriding=true
+####### tomcat is not detecting the x-forward headers from bluemix as a trustworthy proxy
+server.tomcat.internal-proxies=.*
+server.use-forward-headers=true
```

Comment:
Something we missed during our spring-boot-update. This property is now deprecated and replaced with `server.forward-headers-strategy`. Should we create a new ticket to replace this property in all property files + check for further deprecation warnings?